components/switch/test/switch.vdiff.js

```diff
@@ -0,0 +1,50 @@
+import '../switch.js';
+import { expect, fixture, focusElem, hoverElem, html } from '@brightspace-ui/testing';
+
+const offFixture = html`<d2l-switch text="Test Text"></d2l-switch>`;
+const onFixture = html`<d2l-switch text="Test Text" on></d2l-switch>`;
+
+describe('d2l-switch', () => {
+
+	describe('ltr', () => {
```

Comment:
Minor: If you feel like it, I've been removing some places where we had the `ltr` (and then manually removing it from the new golden names). Anywhere where it doesn't really seem to be adding anything 🤷

Reply:
Cool -- I've done this for the switch tests but left the visibility-switch ones since the ltr/rtl suite is unfortunately outside the tests.